modules/candidate_parameters/ajax/getData.php

```diff
@@ -215,12 +215,9 @@ function getFamilyInfoFields()
     }
 
     $familyMembers = $db->pselect(
-        "SELECT CandID as FamilyCandID, Relationship_type 
-        FROM family 
-        WHERE FamilyID=(
-          SELECT FamilyID 
-          FROM family 
-          WHERE CandID = :candid) AND CandID <> :candid2 
+        "SELECT f1.CandID as FamilyCandID, Relationship_type 
```

Comment:
This query cause an error: Column 'Relationship_type' in field list is ambiguous
It should be f1.Relationship_type